Makefile

```diff
@@ -33,14 +35,23 @@ clean:
 	-@ while read -r LINE; do [[ $${LINE} =~ test/.+ ]] && sudo rm -rf $${LINE}; done < .gitignore
 
 # -----------------------------------------------
-# --- Tests  & Lints ----------------------------
+# --- Tests  ------------------------------------
 # -----------------------------------------------
 
-tests:
-	@ ./test/bats/bin/bats --timing test/*.bats
+tests: tests/serial tests/parallel/set1 tests/parallel/set2 tests/parallel/set3
+
+tests/serial:
+	@ ./test/bats/bin/bats --timing --jobs 1 test/$@/**.bats
+
+tests/parallel/set%:
+	@ ./test/bats/bin/bats --timing --jobs $(PARALLEL_JOBS) test/$@/**.bats
 
 test/%:
-	@ ./test/bats/bin/bats --timing $@.bats
+	@ shopt -s globstar nullglob ; ./test/bats/bin/bats --timing test/tests/**/{$*,}.bats
```

Comment:
> I am familiar with this `shopt -s globstar nullglob` line from the change detection scripts. It's just excluding any paths that aren't valid?

Yes

> What's going on with the `**/{$*,}.bats` portion? This is like the `/**.bats` from earlier but if I understand your commit message and example for this (thanks!), it's adding the support to provide multiple files to test instead of just one?

Yes :)

> The example in the commit message was `make test/clamav,template`. So it is `make test/%`, and then we specify the test names (_without serial / parallel folder location, nor `.bats` extension_), and separate with a `,` delimiter if we want to test multiple files?

Exactly; you could actually specify path inside the the `test/tests/` directory as well, that would work too, without the `.bats` extension of course.

> It would be good for the test docs to mention that, especially that the directory path from `test/tests/` to the `.bats` file is not relevant.

Will do

> I assume you can provide additional specificity there, in the event that we had two tests with the same filename at different locations? (_Otherwise it'd run both of those right?_)

Yes